src/gradleinttest.sh

```diff
@@ -37,13 +37,13 @@ gradleinttest () {
     echo "Running ${test_to_run} test with data project env var unset: ${google_data_project}"
   fi
 
-  if [[ -n "${google_project}" ]] && [ -f jade-dev-account.json ] && [ -f jade-dev-account.pem ] && [[ "${test_to_run}" != "" ]]; then
+  if [[ -n "${google_project}" ]] && [[ "${google_application_credentials}" != "" ]] && [ -f ${google_application_credentials} ] && [[ "${google_application_credentials_pem}" != "" ]] && [ -f ${google_application_credentials_pem} ] && [[ "${test_to_run}" != "" ]]; then
     export PGHOST=$(ip route show default | awk '/default/ {print $3}')
     export DB_DATAREPO_URI="jdbc:postgresql://${PGHOST}:5432/datarepo"
     export DB_STAIRWAY_URI="jdbc:postgresql://${PGHOST}:5432/stairway"
-    export GOOGLE_APPLICATION_CREDENTIALS=jade-dev-account.json
-    export IT_JADE_PEM_FILE_NAME=jade-dev-account.pem
-    export GOOGLE_SA_CERT=jade-dev-account.pem
+    export GOOGLE_APPLICATION_CREDENTIALS=${google_application_credentials}
```

Comment:
Why do we have both uppercase and lowercase versions of the credentials? Why not simply define a GOOGLE_APPLICATION_CREDENTIALS in the env?

Reply:
I'm following a convention that is used in the rest of the GitHub action code (see main.sh) where we define the input in lower case and then only export the needed variables (in uppercase) as needed.  I think it would be worth looking into this and reworking our variable definition, but I think it's outside of the scope of this ticket.